planner/core/src/test/java/DamGenTests.java

```diff
@@ -16,18 +16,132 @@
  */
 
 import com.google.common.io.Resources;
+import eu.seaclouds.platform.planner.core.HttpHelper;
 import eu.seaclouds.platform.planner.core.Planner;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
 import org.testng.annotations.Test;
+import org.yaml.snakeyaml.Yaml;
+import sun.net.www.http.HttpClient;
 
 import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Scanner;
 
+import static org.testng.AssertJUnit.assertEquals;
 import static org.testng.AssertJUnit.assertNotNull;
 import static org.testng.AssertJUnit.assertTrue;
 
 public class DamGenTests {
 
     @Test
+    public void damTranslation() throws Exception{
```

Comment:
I don't want to be a problem. I'm simply saying it is not clear and a javadoc could help here.